api/src/main/java/org/motechproject/nms/api/web/UserController.java

```diff
@@ -137,7 +135,7 @@ public UserResponse getUserDetails(@PathVariable String serviceName,
 
         // If the user does not have a language location code we want to return the allowed language location
         // codes for the provided circle, or all if no circle was provided
-        List<Language> languages = new ArrayList<>();
+        Set<Language> languages = new HashSet<>();
         if (user.getLanguageLocationCode() == null && circleObj != null) {
```

Comment:
Unrelated to this, but intellij warns of a NPE here since there is a possibility that the user object was never set.